readme.md

```diff
@@ -14,46 +15,45 @@
 ## Install
 
 ```sh
-npm i --save design-system-utils
-
 yarn add design-system-utils
+
+# or
+
+npm install --save design-system-utils
 ```
 
 ### Size
 
-Package size: **925 B unminified** :+1:
+Package size: **925 B unminified** 👍
 
 ## Usage
 
-You first need to create your design system file, this contains all your global variables that your app will use, think font-sizes, color palette, spacing etc. I usually create a top-level directory named `theme` or `designsystem`, and add an index.js inside, like so:
+You first need to create your design system file, this contains all your design tokens that your app or site will use, think font-sizes, color palette, spacing etc. I usually create a top-level directory named `tokens`, `theme` or `designsystem`, and add an index.js inside, like so:
```

Comment:
would suggest changing the focus of the documentation to the consumer of the code
```diff
- I usually
+ For example you can
create a top-level directory named ...
```

Reply:
on second thoughts ignore this comment ☝️ ❌